common/config/rush/command-line.json

```diff
@@ -15,13 +15,13 @@
       "name": "check-latest-updates",
       "summary": "Upgrade dependencies in all projects to the latest version not already satisfied by the semver range. This includes major versions and is best to run after `check-minor-updates`. You must run `rush update` after.",
       "commandKind": "global",
-      "shellCommand": "node common/scripts/install-run.js npm-check-updates@16.10.15 npm-check-updates --packageFile *{$(common/scripts/get_comma_separated_rush_directories.sh)}/package.json --upgrade --minimal --target latest --reject '@types/node'"
+      "shellCommand": "node common/scripts/install-run.js npm-check-updates@16.14.18 npm-check-updates --packageFile *{$(common/scripts/get_comma_separated_rush_directories.sh)}/package.json --upgrade --minimal --dep 'prod','dev','optional','packageManager','peer' --target latest --reject '@types/node,eslint'"
```

Comment:
how to remember these updates? I guess it doesn't matter as much because no implication of lockfile changes?